src/migration2018/gutenbergblocks.py

```diff
@@ -1976,6 +1992,49 @@ def _fix_epfl_video(self, content, page_id):
 
         return content
 
+    
+    def _fix_pdfjs_viewer(self, content, page_id):
+        """
+        Transforms PDFJS Viewer shortcode to Gutenberg block
+
+        :param content: content to update
+        :param page_id: Id of page containing content
+        """
+        shortcode = 'pdfjs-viewer'
+        block = 'pdf-viewer-block/standard'
+
+        # Looking for all calls to modify them one by one
+        calls = self._get_all_shortcode_calls(content, shortcode)
+
+        for call in calls:
+
+            url = self._get_attribute(call, 'url')
+            viewer_width = self._get_attribute(call, 'viewer_width')
+            viewer_height = self._get_attribute(call, 'viewer_height')
+            
+            attributes = {}
+
+            id = self._get_attachment_id(url)
+
+            if id is None:
```

Comment:
je suis plus enclin à un if not id, par habitude
```suggestion
            if not id:
```